handbook/engineering/career-development/talent-review-process.md

```diff
@@ -0,0 +1,93 @@
+
+## Calibration meeting
+
+The calibration meeting is a several-hour meeting (possibly spread out over two days, depending on logistics) with all of the EMs and directors of the Group, plus an observer from [People Ops](../../people-ops/index.md) for the purpose of reducing unconscious bias. (If the Group is not all of engineering, then at least one other director is invited as well, to help ensure that all Groups are well-calibrated with each other.) Upper management may be invited, but are not required and have no official role.
+
+All ICs at a given level are reviewed together, starting with level 1. Every EM with an IC at level 1 places their ICs into one of these categories: struggling/meeting/exceeding expectations. Then, one at a time, each IC’s placement is explained by their EM. (Note that in the interests of time, the Group might decide to focus conversation on ICs who are under- or over-performing, especially given the async prework described above.) If they are being pitched for a promotion, that is also shared. Other EMs then provide additional feedback/insight they might have, either to challenge or to further justify the IC’s placement. _This is not adversarial:_ the EMs are working together to best establish where each IC is, and to help prepare the EM to provide meaningful feedback to the IC.
+
+During the discussion, there is a rotating _moderator_ whose job it is to ensure that a fair, balanced, and productive conversation is happening. This can include asking questions of the other participants, calling out potential biases, and calling “time” when things are going in circles.
+
+After all level 1 ICs have been discussed, we move on to level 2 ICs. For any IC being pitched for a promotion to level 2, they are evaluated along with their prospective cohort. (They are not discussed again in the same way, but after discussing other ICs with the same meeting/exceeding evaluation, a quick check is done to validate that they are performing similarly to those folks at level 2.)
+
+This is done for each of our engineering levels.
+
+## After the meeting
+
+Decisions regarding promotions are prioritized by the relevant decision-makers (EM, Director, VP), and made quickly. Promotions are communicated back to the ICs in a timely manner. Regardless of promotions, any qualitative feedback for an IC that came out of talent review is shared with them in their next one-on-one (if not sooner).
+
+It’s important that this process moves quickly! If we leave people wondering for a month or more if they will be getting a promotion, we’re creating needless anxiety and uncertainty.
+
+Finally, we should have a [retrospective](../../retrospectives/index.md) after each of these, so we can continue iterating and improving upon this process. We should also consider changes/clarifications to the [level descriptions](framework.md#levels) at this time, in case any of the existing definitions were cause for confusion.
+
+## Levels
```

Comment:
much of the content in this section feels more relevant to the page that documents the levels themselves rather than the talent review process. There is also some duplication that I think would be nice to remove.

Reply:
I will remove some of the duplication tomorrow, but I think we might merge what we have first, before Felix goes on break.

Reply:
Thanks for the feedback! @chrispine will do some deduplication tomorrow